cmd/apis/oascrtapisv2.go

```diff
@@ -0,0 +1,215 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package apis
+
+import (
+	"fmt"
+	"os"
+
+	"internal/apiclient"
+	"internal/clilog"
+
+	bundle "internal/bundlegen"
+	proxybundle "internal/bundlegen/proxybundle"
+
+	"internal/client/apis"
+
+	"github.com/spf13/cobra"
+)
+
+var OasCreatev2Cmd = &cobra.Command{
+	Use:     "openapi",
+	Aliases: []string{"oas"},
+	Short:   "Creates an API proxy from an OpenAPI Specification",
+	Long:    "Creates an API proxy from an OpenAPI Specification",
+	Args: func(cmd *cobra.Command, args []string) (err error) {
+		if oasFile == "" && oasURI == "" {
+			return fmt.Errorf("either oasfile or oasuri must be passed")
```

Comment:
Incorrect error message, should be:
either oas-base-folderpath or oas-base-uri must be passed